packages/react/__tests__/exports.ts

```diff
@@ -1373,6 +1373,7 @@ describe('@aws-amplify/ui-react', () => {
           "ScrollView",
           "SearchField",
           "SelectField",
+          "SharedText",
```

Comment:
@reesscot This should still be removed, right?

Reply:
yea, that should be removed for sure

Reply:
Yup, I can add the removal back in followup pr